package.json

```diff
@@ -53,6 +53,7 @@
     "@next/mdx": "^14.1.0",
     "@octokit/graphql": "^7.0.2",
     "@slack/web-api": "^7.0.2",
+    "@storybook/preview-api": "^8.1.11",
```

Comment:
why we need this library?

Reply:
We need this to use [useArgs](https://storybook.js.org/docs/addons/addons-api#useargs) hook that allows you to retrieve or update a story's [args](https://storybook.js.org/docs/writing-stories/args).